ui/src/lib/interfaces.ts

```diff
@@ -500,3 +500,18 @@ export interface TopicsPaginatedResults {
 export interface TopicsPageQuery extends ParsedUrlQuery {
     query?: string;
 }
+
+export interface TopicsPaginatedResults {
```

Comment:
This is a duplicate. It was already defined on line 489. It might come from rebase.

Reply:
Now removed.